circle.yml

```diff
@@ -49,6 +53,8 @@ test:
     - cp -R ./* $DRUPAL_SITE_PATH/profiles/ding2/
     # Install the site using the ding2 profile
     - (cd $DRUPAL_SITE_PATH && drush site-install ding2 --db-url=mysql://ubuntu@127.0.0.1/circle_test -y)
+    # Run PHP7 Compatibility Checker
+    - 'php7cc --except=vendor --level=error --extensions=php,inc,module,install $DRUPAL_SITE_PATH | tee /dev/tty | grep -vq ''File: '''
```

Comment:
Adding a comment about why we invoke php7cc as we do would be good. Consider adding a reference to [the related issue](https://github.com/sstalle/php7cc/issues/102).

Reply:
Added